eclipse-collections-code-generator/src/main/resources/impl/map/immutable/primitiveImmutableKeySets.stg

```diff
@@ -6,7 +6,7 @@ import "primitiveLiteral.stg"
 primitiveKeySetBody(className, type, name, keyArray, increment, sameTwoPrimitiveMapKeySet, calledFrom) ::= <<
 <copyright()>
 
-package org.eclipse.collections.impl.map.mutable.primitive;
+package org.eclipse.collections.impl.map.immutable.primitive;
```

Comment:
This cannot be moved as it will break serialization.

Reply:
reverted